server/cpho/jinja2/indicators/upload_indicator.jinja2

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.jinja2' %}
+{% block content %}
+  <form action="{{ url('upload_indicator') }}"
```

Comment:
render form errors here, i think `{{ form.errors }}` will be good enough to develop, you can style it later by accessing it differently